addons/web/static/src/legacy/js/core/utils.js

```diff
@@ -1140,9 +1140,39 @@ var utils = Object.assign({
             ['res_model', '=', 'ir.ui.view'],
             '|',
             '|',
+            '|',
+            '|',
+            '|',
+            '|',
+            '|',
+            '|',
+            '|',
+            '|',
+            '|',
+            '|',
+            '|',
+            '|',
+            '|',
+            '|',
+            '|',
```

Comment:
May I suggest that you generate all these `'|', ['name', '=like', <pattern>]` from a list of patterns defined separately? when there are only 3 entries it's pretty easy to see at a glance that there are the right number of matching '|' and [...], but with this many, it becomes error prone. As a bonus, if you define this list of assets patterns directly on the utils object, other modules could inject their own patterns if they generate assets with weird names. (and maybe that's what POS and spreadsheet should be doing)

Reply:
Created enterprise PR https://github.com/odoo/enterprise/pull/26681